java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -73,4 +73,91 @@ public interface BitmapBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> bitcount(String key, long start, long end, BitmapIndexType options);
+
+    /**
+     * Return the position of the first bit set to 1 or 0 in a string.
+     *
+     * @see <a href="https://redis.io/commands/bitpos/">redis.io</a> for details.
+     * @param key The key for the string to count the set bits of.
+     * @param bit The bit value for finding first occurrence.
+     * @return The position of the first occurrence of the <code>bit</code> in the binary value of the
+     *     string held at <code>key</code>.
+     * @example
+     *     <pre>{@code
+     * Long payload = client.bitpos("myKey1", 0).get();
+     * assert payload == 3L; // Indicates that the first occurrence of a 0 bit value of the string stored at "myKey1" is at the fourth element.
+     * }</pre>
+     */
+    CompletableFuture<Long> bitpos(String key, long bit);
+
+    /**
+     * Return the position of the first bit set to 1 or 0 in a string. The offsets <code>start</code>
+     * and <code>end</code> are zero-based indexes, with <code>0</code> being the first element of the
```

Comment:
```suggestion
     * is a zero-based indexes, with <code>0</code> being the first byte of the
```
note: by default, the start and end positions are specified as BYTE